lib/helpers.js

```diff
@@ -61,15 +62,16 @@ async function getAndroidPlatformAndPath () {
   return result;
 }
 
-async function unzipFile (zipPath) {
-  log.debug(`Unzipping ${zipPath}`);
+async function unzipFile (zipPath, dstRoot = null) {
+  dstRoot = dstRoot || path.dirname(zipPath);
```

Comment:
Is it ok to use other arguments in default value calculation?

Reply:
Yes, as long as they are later in the argument list. So
```js
function a (b = c, c) {
  console.log(b, c);
}
```
would fail if you called `a(undefined, 1)`, with a reference error, because `c` would not have been defined when `b` is initialized to it.

But, for every situation, the follow is fine:
```js
function a (b, c = b) {
  console.log(b, c);
}
```

Reply:
ok, fixed